lib/no_ssn_validator.js

```diff
@@ -154,6 +154,40 @@
 
       return checksum;
     };
+
+    exports.getCentury = function (individualNumber, birthYear) {
+      var century;
+      if (individualNumber >= 0 && individualNumber <= 499) {
+        century = 19;
+      } else if (individualNumber >= 500 && individualNumber <= 749 && birthYear >= 55) {
+        century = 18;
+      } else if (individualNumber >= 500 && individualNumber <= 999 && birthYear <= 39) {
+        century = 20;
+      } else if (individualNumber >= 900 && individualNumber <= 999 && birthYear >= 40) {
+        century = 19;
+      }
+      return century;
+    };
+
+    exports.getBirthDate = function (ssn) {
+      if(!exports.isValid(ssn)) {
+        return false;
```

Comment:
This makes the function return two different types. Could we change this to return `null` or `undefined`.